projects/wave-core-new/src/public-api.ts

```diff
@@ -55,6 +55,10 @@ export * from './lib/time/small-time-interaction/small-time-interaction.componen
 export * from './lib/time/time-config/time-config.component';
 export * from './lib/time/time-input/time-input.component';
 export * from './lib/users/login/login.component';
+export * from './lib/operators/dialogs/helpers/multi-layer-selection/multi-layer-selection.component';
+export * from './lib/operators/dialogs/helpers/operator-output-name/operator-output-name.component';
+export * from './lib/operators/dialogs/operator-list/operator-list.component';
+export * from './lib/operators/dialogs/expression-operator/expression-operator.component';
```

Comment:
It sounds like OCD, but could you sort these lines after adding new stuff? It is super complicated to get an overview over the exporter files this way.

Reply:
81fdbc6ddab0a5a01da03f91b42ba9356982f9d0